JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl

```diff
@@ -1,22 +1,43 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl#"
-     xml:base="https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl"
-     xmlns:OntoTimeSeries="https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl#"
-     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
```

Comment:
I know that the updated base IRI `https://www.theworldavatar.com/kg/ontotimeseries/` follows the new convention; however, that would mean that most of the current agents using the time series with the "old" base IRI `https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl` would become erroneous. I think we might keep the old base IRI for now.
@sm453 What is your opinion on that? Maybe we could also use a different version of the ontology to avoid this issue, but I am not too sure how that would work exactly (especially with referencing the right version when instantiating data)

Reply:
This is frustrating.

While IRIs are completely arbitrary in general, something like https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl shouldn't really be used for anything other than development/testing purposes. From that point of view, I'd prefer a correct production IRI https://www.theworldavatar.com/kg/ontotimeseries/, although I appreciate that this doesn't play well with some tools that some people currently like to use.

More fundamentally, people shouldn't be using OWL files at all (at least not for this purpose), but I appreciate that at present there is no alternative in place.

Consistency across the code base is a practical issue, however a temporary one that will go away once we start doing things properly (which is admittedly not going to happen any time soon).

So, unfortunately, I have to agree, very reluctantly and under protest, that this awful IRI https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl may be the path of least resistance for now.

Reply:
>From that point of view, I'd prefer a correct production IRI https://www.theworldavatar.com/kg/ontotimeseries/, although I appreciate that this doesn't play well with some tools that some people currently like to use.

It's not that those tools, i.e. instantiation scripts and agents, would not work anymore. It's just that they instantiate "outdated", i.e. meaningless, IRIs.

> So, unfortunately, I have to agree, very reluctantly and under protest, that this awful IRI https://github.com/cambridge-cares/TheWorldAvatar/blob/main/JPS_Ontology/ontology/ontotimeseries/OntoTimeSeries.owl may be the path of least resistance for now.

I do remember that we had exactly this discussion about the `timeseries` IRI a few months back and decided to keep it "for now" as this IRI is hardcoded in the TimeSeriesClient and would have required quite a few changes. Another perspective to look at it is the follows:
Mehal just pushed and merged changes to the TimeSeriesClient and we could easily add a hotfix and change the base IRI. Jiaru just release a new py4jps version and we could also add a hotfixed version there.
The stack (for King's Lynn) is still not working and I have to (re-)deploy all agents from scratch anyway. Yi Kai and Magnus are going to instantiate time series data soon as well. So, the cost of switching are just getting bigger and it might actually be not a bad moment to implement those changes now as at least 3 people are working with the updated version then.
@sm453 Any thoughts?

Reply:
I agree. Go for it - it's not getting any better...